### PR TITLE
Add table `teamkills` for server-issue #95

### PR DESCRIPTION
**Pull request by Brutus5000**
Prerequisite to work on https://github.com/FAForever/server/issues/95
Table structure is based upon the new GPGNet-command from https://github.com/FAForever/fa/pull/903

### CODE REVIEW

**Comment by Crotalus** [comment]
Maybe add a reference to what game it happened as well if someone wants to investigate it more carefully

**Comment by Brutus5000** [comment]
Good idea, done.

**Comment by Brutus5000** [comment]
There was a slight misunderstanding with the GPGNet-Command on my side. The killtime is the time in seconds of game running and not the local time, therefore we should save this in the db as well.

File: migrations/add_teamkills-ab3d56fe949550d2ac113a148582e4c7c690e25c.sql [hunk 1]
```diff
@@ -0,0 +1,6 @@
+CREATE TABLE IF NOT EXISTS `teamkills` (
+  `teamkiller` mediumint(8) unsigned NOT NULL COMMENT 'login of the player who performed the teamkill',
+  `victim` mediumint(8) unsigned NOT NULL COMMENT 'login of the player who got teamkilled and reported the tk',
+  `game` int(10) unsigned NOT NULL COMMENT 'game-id where teamkill was performed',
+  `gametime` mediumint(8) unsigned NOT NULL COMMENT 'time of game in seconds when tk was performed',
+  `reported_at` datetime NOT NULL DEFAULT CURRENT_TIMESTAMP);
```
Comment (Sheeo) [inline migrations/add_teamkills-ab3d56fe949550d2ac113a148582e4c7c690e25c.sql]: I'm being nitpicky here (It's a rather small problem), but please configure your editor to insert trailing newlines at EOF.

This matters for certain CLI tools :)

**Comment by Sheeo** [comment]
It's good for a first -- I suggest we add an index on `game` (Shouldn't we call this column `game_id`?) immediately though, since that's probably what we're going to be querying on.

Also, since I just merged Dragonfire's autogenerated db-structure.sql file, would you mind rebasing this onto the latest develop? You'll want to run Dragonfire's regenerate script.

**Comment by Brutus5000** [comment]
Rebased and all suggestions implemented.

**Comment by Sheeo** [comment]
👍

**Comment by Brutus5000** [comment]
Connects to #95 (waffle.io reference)

**Pull request merged**